apps/builddao/widget/Compose.jsx

```diff
@@ -414,28 +451,52 @@ const avatarComponent = useMemo(() => {
   );
 }, [context.accountId]);
 
+const editorKey = `Editor-Content-${currentTemplate.title}`
+
+const textEditorComponent = useMemo(() => {
+  return (
+    <Widget
+      key={editorKey}
+      src="buildhub.near/widget/components.Text.Editor"
+      props={{
+        initialText: postContent || "What's happening?",
+        embedCss: MarkdownEditor,
+        onChange: (v) => {
+          setPostContent(v);
+          Storage.privateSet(draftKey, v || "");
+        },
+      }}
+    />
+  )
+}, [currentTemplate])
+
 return (
   <PostCreator>
     {avatarComponent}
+    <Widget 
```

Comment:
This widget is very intense -- it slows down the feed a lot.

We need to resolve this before merge